experimental/sql/src/impl_to_iterators.py

```diff
@@ -338,6 +353,16 @@ def get_batch_and_name_list(
           ]))
       batches.append(curr_batch)
       names.append(o.table_name.data)
```

Comment:
I think the documentation of this function (`Scans over all the operations ...`) should be updated to explain what happens with partial scans/projections.

Reply:
This comment is out of date, right?